pyserini/util.py

```diff
@@ -150,8 +154,11 @@ def check_downloaded(index_name):
     return os.path.exists(index_path)
 
 
-def get_indexes_info():
-    df = pd.DataFrame.from_dict(INDEX_INFO)
+def get_indexes_info(dense=False):
```

Comment:
How about two separate methods to be explicit? E.g., `get_dense_indexes_info` and `get_sparse_indexes_info`?

Seems cleaner than this optional arg?

Reply:
yeah, sure